docs/edit_first_game_server.md

```diff
@@ -1,11 +1,15 @@
 # Getting Started
-The following guide is for developers without Docker or Kubernetes experience, that want to use the simple-udp example as a starting point for a custom game server. Since this guide is for Google Kubernetes Engine, we welcome a Pull Request to expand this to include Minikube as well.
+The following guide is for developers without Docker or Kubernetes experience, that want to use the simple-udp example as a starting point for a custom game server. This guide addresses Google Kubernetes Engine and Minikube.  We would welcome a Pull Request to expand this to include other platforms as well.
 
 ## Prerequisites
 
 1. Downland and install Golang from https://golang.org/dl/.
 2. Install Docker from https://www.docker.com/get-docker.
-3. Follow the install instructions (if you haven't already) at [Install and configure Agones on Kubernetes](../install/README.md). At least, you should have "Setting up a Google Kubernetes Engine (GKE) cluster", "Enabling creation of RBAC resources" and "Installing Agones" done.
+3. Install Agones on GKE or Minikube.  
+
+To install on GKE, follow the install instructions (if you haven't already) at [Install and configure Agones on Kubernetes](../install/README.md). At least, you should have "Setting up a Google Kubernetes Engine (GKE) cluster", "Enabling creation of RBAC resources" and "Installing Agones" done.
+
+To install locally on Minikube, read [Developing, Testing, and Building Agones](../build/README.md) making sure to follow the instructions in the [Running A Test Minikube cluster](../build/README.md#running-a-test-minikube-cluster) section.
```

Comment:
Should we link to installing on Minikube instead?
https://github.com/GoogleCloudPlatform/agones/blob/master/install/README.md#setting-up-a-minikube-cluster

Reply:
Pros seem to be that it is less complex and requires less reading time.  
Cons are that the reader is left to discover build tools on their own, and they might need information about having the project on their gopath.

Reply:
So I would argue that if they are developing with Agones - they shouldn't need the build tools - they should install and use Agones as per normal.

I'm very hesitant to have the streams of using Agones and Developing Agones mix - they have very different targets, and it could get super messy in the long term.

Reply:
Per your suggestion, this now links to https://github.com/GoogleCloudPlatform/agones/blob/master/install/README.md#setting-up-a-minikube-cluster